example/64_complex_contraction_bilinear/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_example_executable(example_complex_contraction_bilinear_xdl_fp32 complex_contraction_bilinear_xdl_fp32.cpp)
```

Comment:
Looks like example 64 is already used, could you assign a different number?

Reply:
I made it example 66